bundles/org.eclipse.jface/src/org/eclipse/jface/widgets/AbstractWidgetFactory.java

```diff
@@ -102,4 +102,21 @@ public F data(Object data) {
 		addProperty(b -> b.setData(data));
 		return cast(this);
 	}
+
+	/**
+	 * Sets the application defined property of the receiver with the specified name
+	 * to the given value.
+	 *
+	 * @param key   the name of the property
+	 * @param value the new value for the property
+	 * @return this
+	 *
+	 * @see Widget#setData(String, Object)
+	 *
+	 * @since 3.34
```

Comment:
Error:  Failed to execute goal org.eclipse.tycho:tycho-apitools-plugin:4.0.5-SNAPSHOT:verify (verify) on project org.eclipse.jface: There are API errors:
Error:  src/org/eclipse/jface/widgets/AbstractWidgetFactory.java:117 Invalid @since 3.34 tag on data(String, Object); expecting @since 3.33